src/gsy_e/models/area/__init__.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, name: str = None,
         self.uuid = uuid if uuid is not None else str(uuid4())
         self.slug = slugify(name, to_lower=True)
         self.parent = None
+        if not children:
+            children = []
+        children = [child for child in children if child is not None]
         self.children = (
             AreaChildrenList(self, children)
             if children is not None
```

Comment:
I think that we should change this to align with the change:
```suggestion
            if children
```
There is also a children set to None in the [area_serializer](https://github.com/gridsingularity/gsy-e/blob/6774e3819af1b17405beca31a1751eff45cfcbd5/src/gsy_e/gsy_e_core/area_serializer.py#L148) , so maybe that part should be aligned as well (not sure about it).

Reply:
I wanted to catch all cases (children == None, children == [], children == {}) in order to harmonize all cases and to not fail, hence the use of the `not`. In a better implementation, it should be inside the AreaChildrenList class, but since we are in a hurry I wanted to go the easy way out.